File: competitive_programming/codeforces/01_python/617a_elephant.py
"""Codeforces 617A - Elephant."""
def elephant(position):
    return (position + 4) // 5


if __name__ == "__main__":
    assert elephant(5) == 1
    assert elephant(12) == 3
    assert elephant(1) == 1
    print("617A elephant ok")

File: competitive_programming/codeforces/09_rust/617a_elephant.rs
fn elephant(position: i32) -> i32 {
    (position + 4) / 5
}

fn main() {
    assert_eq!(elephant(5), 1);
    assert_eq!(elephant(12), 3);
    assert_eq!(elephant(1), 1);
    println!("617A elephant ok");
}
